modules/core/src/mathfuncs_core.cpp

```diff
@@ -373,6 +373,62 @@ void sqrt64f(const double* src, double* dst, int len)
         dst[i] = std::sqrt(src[i]);
 }
 
+// Workaround for ICE in MSVS 2015 update 3 (issue #7795)
+// CV_AVX is not used here, because generated code is faster in non-AVX mode.
+// (tested with disabled IPP on i5-6300U)
+#if (defined _MSC_VER && _MSC_VER >= 1900)
```

Comment:
Sorry, you need to build with OpenCL (only "OCL" (read as "UMat") tests measure performance for exp/log and [they are disabled if there no OpenCL](https://github.com/opencv/opencv/blob/300f92397e93fab638944630b37257a4e2f4a616/modules/core/test/ocl/test_arithm.cpp#L47)). You need to disable OpenCL during runtime: set environment variable `OPENCV_OPENCL_DEVICE=disabled`.

Also you can try to replace mentioned line to `#if 1` temporary and build without OpenCL.